README.md

```diff
@@ -1,2 +1,4 @@
 # testyMcTestFace
-dsgdsgfgs
+
+
+A proper desc
```

Comment:
/ticket Write something here -d and smething else here

Reply:
Here's your ticket: [ Write something here](https://ticketgen19.atlassian.net/browse/IKKI-55)

Reply:
Here's your ticket: [ Write something here](https://ticketgen19.atlassian.net/browse/IKKI-56)

Reply:
/ticket A nice title -d a nice desc

Reply:
Here's your ticket: [ A nice title](https://ticketgen19.atlassian.net/browse/IKKI-57)

Reply:
Here's your ticket: [ A nice title](https://ticketgen19.atlassian.net/browse/IKKI-58)